src/js/App/Header/HeaderAlert.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import { Alert } from '@patternfly/react-core/dist/js/components/Alert/Alert';
+import { AlertActionCloseButton } from '@patternfly/react-core/dist/js/components/Alert/AlertActionCloseButton';
+import PropTypes from 'prop-types';
+import './HeaderAlert.scss';
+
+const HeaderAlert = ({ title, variant, onDismiss, dismissable, dismissDelay }) => {
+  const [alertVisible, setAlertVisible] = useState(true);
+  const [timer, setTimer] = useState(null);
+
+  useEffect(() => {
+    dismissable || createTimer();
+  }, []);
+
+  useEffect(() => {
+    return () => clearTimeout(timer);
+  }, []);
```

Comment:
You can merge these two together.
```suggestion
  useEffect(() => {
    dismissable || createTimer();
    return () => clearTimeout(timer);
  }, []);
```